BraintreeDataCollector/src/main/java/com/braintreepayments/api/DataCollector.java

```diff
@@ -117,4 +117,32 @@ private String getPayPalClientMetadataId(Context context, Configuration configur
         }
         return "";
     }
+
+    /**
+     * Collects device information for fraud identification purposes from PayPal only.
+     *
+     * @param context           Android Context
+     * @param riskCorrelationId Correlation id to associate with data collection
+     * @param callback          {@link DataCollectorCallback}
+     */
+    public void collectPayPalDeviceData(@NonNull final Context context, @NonNull final String riskCorrelationId, @NonNull final CollectPayPalDeviceDataCallback callback) {
```

Comment:
Just a comment, but we should finalize what we want to call this for next major version on iOS and Android. We call it `pairingID` but later it's called `clientMetadataID` on iOS and here we are using `riskCorrelationId`. Would be nice to align the function signatures a bit more.

Reply:
Yeah on iOS I noticed `collectDeviceData` isn't aware of the Braintree configuration. On Android, we need to fetch config asynchronously before we can return a `clientMetadataId`.

I wonder if there's an opportunity to make it env aware on iOS?

Reply:
+1 that we really need to align on the name of this ID in the next major versions!

`riskCorrelationId` seems to be the clearest and approved as a public term by the risk team.

Reply:
@sshropshire - passing in APIClient to the data collector is in next major version, so we should be aligned there as well. 😄 

@demerino will add a ticket for updating the name on iOS to `riskCorrelationId` in next major version.